detox/src/android/core/NativeElement.js

```diff
@@ -42,12 +42,31 @@ class NativeElement {
     return await new ActionInteraction(this._invocationManager, this._matcher, action, traceDescription).execute();
   }
 
-  async longPress() {
-    const action = new actions.LongPressAction();
-    const traceDescription = actionDescription.longPress();
+  async longPress(arg1, arg2) {
```

Comment:
In case we decide to leave things as they are (see my comment about a single complex parameter with named options), I'd suggest naming them as:

```js
async longPress(maybePoint, maybeDuration) {
  const duration = maybeDuration != null ? maybeDuration : maybePoint;
  const point = maybeDuration != null ? maybePoint : null;
  
  // ... and then you validate the types of `point` and `duration`.
```

Reply:
Regarding https://github.com/wix/Detox/pull/4475#discussion_r1584330256
`longPress(duration)` was already supported in iOS so we won't break this API, also I prefer to extend it in a flat way as we're doing with the rest of our Action APIs

Reply:
It doesn't make sense that `maybePoint` can also be `duration`..
I'll call them `optionalPointOrDuration` and `optionalDuration`.